x-pack/plugins/osquery/public/action_results/translations.ts

```diff
@@ -20,3 +20,24 @@ export const FAIL_ACTION_RESULTS = i18n.translate(
     defaultMessage: `Failed to fetch action results`,
   }
 );
+
+const platinumLicenseRequired = 'At least Platinum license is required to use Response Actions.';
+const parametersNotFound =
+  "This query hasn't been called due to parameter used and its value not found in the alert.";
+
+export const getSkippedQueryError = (error: string) => {
```

Comment:
can you add a return type? based on what I see in the code below, I think it should be `string | null`

Reply:
After applying the comment from above, this function got redundant :)